class AlreadySubscribedException(Exception):
    pass

class NotSubscribedException(Exception):
    pass